src/sentry/static/sentry/app/views/organizationGroupDetails/actions.jsx

```diff
@@ -144,14 +144,12 @@ const GroupDetailsActions = createReactClass({
     const discoverQuery = {
       id: undefined,
       name: group.title || group.type,
-      fields: ['title', 'url', 'count(id)', 'project', 'last_seen'],
-      widths: [400, 200, -1, 200, -1],
-      orderby: '-count_id',
+      fields: ['title', 'release', 'environment', 'user', 'timestamp'],
```

Comment:
@bowencai8 @markstory I've updated the fields here. Is this the preferred order?

Reply:
Works for me.